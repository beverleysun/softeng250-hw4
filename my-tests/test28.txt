8
1 7
7 6
6 5
5 4
4 3
3 2
2 1
0 1
2 0
0 3
4 0
0 5
0 6
7 0
