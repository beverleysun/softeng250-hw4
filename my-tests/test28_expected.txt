
1.75 1.75
Cycle(s):
0 1 7
No
